src/firefly/js/drawingLayers/Artifact.js

```diff
@@ -0,0 +1,198 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+
+
+import {take} from 'redux-saga/effects';
+import {get} from 'lodash';
+import {isPlotIdInPvNewPlotInfoAry,
+         isDrawLayerVisible, getDrawLayerById, getPlotViewById} from '../visualize/PlotViewUtil.js';
+import {getRelatedDataById} from '../visualize/RelatedDataUtil.js';
+import {clone, logError} from '../util/WebUtil.js';
+import ImagePlotCntlr, {visRoot} from '../visualize/ImagePlotCntlr.js';
+import PointDataObj, {DrawSymbol} from '../visualize/draw/PointDataObj.js';
+import {makeDrawingDef, getNextColor} from '../visualize/draw/DrawingDef.js';
+import DrawLayer, {ColorChangeType} from '../visualize/draw/DrawLayer.js';
+import {makeFactoryDef} from '../visualize/draw/DrawLayerFactory.js';
+import DrawLayerCntlr, {dlRoot, dispatchModifyCustomField} from '../visualize/DrawLayerCntlr.js';
+import {makeWorldPt} from '../visualize/Point.js';
+import {doFetchTable} from '../tables/TableUtil.js';
+import {getUIComponent} from './CatalogUI.jsx';
+import {dispatchAddSaga} from '../core/MasterSaga.js';
+import {getCenterColumns} from '../tables/TableInfoUtil.js';
+
+const TYPE_ID= 'ARTIFACT_TYPE';
+
+const factoryDef= makeFactoryDef(TYPE_ID,creator,null, getLayerChanges,null,getUIComponent);
+export default {factoryDef, TYPE_ID}; // every draw layer must default export with factoryDef and TYPE_ID
+
+const defSymbolMap= {
+    pers_arti  : {color: 'orange', symbol: DrawSymbol.CROSS, size: 3},
+    glint_arti : {color: 'purple', symbol: DrawSymbol.DIAMOND, size: 3},
+    diff_spikes: {color: 'orange', symbol: DrawSymbol.DOT, size: 1},
+    halos : {color: 'yellow', symbol: DrawSymbol.SQUARE, size: 3},
+    ghost : {color: 'pink', symbol: DrawSymbol.DIAMOND, size: 3},
+    latents : {color: 'green', symbol: DrawSymbol.X, size: 3},
```

Comment:
suggestion: green might be difficult for some viewer, how about 'cyan' or some color neither green nor red.

Reply:
I was just matching the color from the old UI.  It is a good suggestion.  I will change it.